### ISSUE TITLE
See summary of update conflicts on Entity Detail page

### ISSUE THREAD

**Issue by matthew-white** [getodk/central#505]
See the story in the [release criteria](https://docs.google.com/document/d/1EdEDcXoJmrryFYNc56zciuQ82G_iVGBQf1UFEkWiUtw/edit#heading=h.970ohswy1cuc).

---

Related:

- #511
- getodk/central-backend#1028
- getodk/central-backend#1038
- #530
- #529

**Comment by issa-tseng** [comment]
same first-column width issue in the table as the other story

and also the columns should be a _minimum_ of 50px, not _always_ 50px

**Comment by dbemke** [comment]
From the release criteria: "When I click on the source name, the page should scroll to that entry in the feed" . Does this rule apply only if I click the source name in the conflicts table? If I click the version in "Data updated by submission.... v .." the page also scroll to this feed (present in the URL and outlined blue).
Example in the video:
https://jam.dev/c/baa0ebe8-ea61-4d4d-ab65-ca8bf5041255

**Comment by dbemke** [comment]
After clicking the source name, the page scroll to the entry in the feed (+blue outline). After that if I click e.g. a tab in a different parallel update feed (without clicking on the version), should the bkue outline and scrolling to the place be removed? If not, in which cases it should be removed (e.g. clicking different source in the table with conflicts, editing entity)?
Example in the video:
https://jam.dev/c/001efffa-e0c1-4854-a903-9cedc4ce9180

**Comment by dbemke** [comment]
The green check icon for the known good version doesn't appear in the table of conflicts after resolving conflicts. After resolving conflicts and creating new ones all conflict are shown in the table
Steps to reproduce:
1. Create a conflicts and resolve it.
2.  Update the entity from the most current version (Collect or Enketo).
3. Create another conflict.
4. Check what is the last known good version.
I used this entity and trees update form in this project:
https://staging.getodk.cloud/#/projects/20/entity-lists/trees/entities/0e1c22f4-9345-42a0-8da3-e493514843fe

**Comment by matthew-white** [comment]
> The green check icon for the known good version doesn't appear in the table of conflicts after resolving conflicts. After resolving conflicts and creating new ones all conflict are shown in the table

OK, I think we've fixed this one now. Would you mind taking a look?

**Comment by dbemke** [comment]
The last known good version is fixed

**Comment by dbemke** [comment]
I found an example of a conflict which is mark a soft conflict but looking at the version I think it should be a hard conflict (v37-v44)? The properties didn't change but the update (v37) was made on out of date version.
I used a Tree update form without required questions and left the updated properties empty (via Collect). I think after second time I left the property empty the grayed out info about the previous property disappeared
https://staging.getodk.cloud/#/projects/20/entity-lists/trees/entities/0e1c22f4-9345-42a0-8da3-e493514843fe#v41
